internal/repositories/vehicle/vehicles.go

```diff
@@ -249,20 +271,21 @@ func ToAPI(v *models.Vehicle, imageURL string, dataURI string) (*gmodel.Vehicle,
 		Owner:    common.BytesToAddress(v.OwnerAddress),
 		MintedAt: v.MintedAt,
 		Definition: &gmodel.Definition{
-			URI:   v.DefinitionURI.Ptr(),
+			ID:    v.DeviceDefinitionID.Ptr(),
 			Make:  v.Make.Ptr(),
 			Model: v.Model.Ptr(),
 			Year:  v.Year.Ptr(),
 		},
 		ManufacturerID: v.ManufacturerID,
 		Name:           name,
-		Image:          imageURL,
+		ImageURI:       imageURI,
+		Image:          imageURI,
```

Comment:
should these both be the same?

Reply:
Yes. We want to get rid of `image` but mobile are using it.

Reply:
ah, got it... maybe we can add a comment that says `// in the process of deprecating this, mobile is still using it` or something similar

Reply:
See https://github.com/DIMO-Network/identity-api/blob/d53a4bff4ea0ede8c9fd2095db424c8a2ddcf920/graph/schema/vehicle.graphqls#L130